.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - uses: actions/setup-node@v3
+    - uses: actions/setup-node@v4
       with:
         node-version: '16'
```

Comment:
will be worthy to update to node 18 in another iteration as node 16 reached End Of Life in September